teach_base/lesson_02/04_vowels_hash.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Fill the hash with vowels, where the value will be ordinal
+# letter number in the alphabet (a - 1).
+
+vowels_arr = %w[a e i o u y]
+vowels_hash = {}
+
+('a'..'z').each_with_index do |value, index|
+  vowels_hash [index + 1] = value if vowels_arr.include? value
```

Comment:
It's the second time you confuse keys and values.
The letter should be the key, and it's position in alphabet is should be the value.

Don't use the attributes without () brackets!
Don't use spaces before [] brackets!